lib/server/Server.js

```diff
@@ -19,6 +20,7 @@ var Server = {
         var onRouteHandled = config.onRouteHandled;
         var ServerAppToUse = config.ServerApp || ServerApp;
         var appRoot = environmentConfig.appRoot;
+        var proxyClientRequestsToApi = config.proxyClientRequestsToApi;
```

Comment:
I wouldn't make this the default behavior of Brisket because it will cause confusion for people who do not plan to use Brisket that way. I'm not sure if it is an intuitive behavior for those who are not familiar with Brisket. Then, it makes sense to let the user pass in that config to indication their intention.